### DISCUSSION TITLE
Subscribe to record, but also fetch it once

### DISCUSSION THREAD

**Discussion by VilleKokkarinen** [pocketbase/pocketbase#2144] [Q&A]
Is there a existing feature that would combine both of these into one request?
(subscribe to changes, but also give the record back on first fetch, instead of waiting for changes to happen.)

```
 observeChanges(id: string): Observable<Record> {
    return new Observable((observer) => {
      try{
        this.collection.subscribe<Record>(id, function(data){
          observer.next(data.record)
        })
      }catch(err){
        observer.error(err);
      }
    })
  }

  getOne(id: string): Promise<Record>{
    return this.collection.getOne<Record>(id);
  }
```

**Comment by ganigeorgiev** [comment]
Something like this was requested several times already in the past, but for now remains out of the scope since it is not clear what the behavior should be in case a wildcard topic (`*`) is used.
